index.html

```diff
@@ -1381,20 +1381,20 @@
 
 This right includes both being able to review what information has been collected or inferred about
 oneself and being able to discover what [=actors=] have collected information about oneself. As a
-result, databases cannot be kept secret and data collected about [=people=] needs to be meaningfully
+result, databases containing information about [=people=] cannot be kept secret and data collected about people needs to be meaningfully
```

Comment:
A minor tweak on top of yours:
```suggestion
result, databases containing information about [=people=] cannot be kept secret, and data collected about people needs to be meaningfully
```